### PR TITLE
fix(Core/Gameobject): Towercanno error location in Wintergrasp

### PR DESCRIPTION
**Pull request by pvp100com**
## Changes Proposed:
-  adjusted wintergrasp "Flamewatch Tower" towercanno position

## Issues Addressed:

-  fix towercanno error location

## Tests Performed:

- It builds without errors
- Tested in-game

- Server:
AzerothCore rev. 893a9beeacc4+
Debian 11 x64
MySQL 8.0.33

- Client:
3.3.5a.12340 enUS
Windows 10 x64 LTSC

## How to Test the Changes:

- check towercanno error location: 
.go xyz 4448.713379 1955.148682 441.995178 571
-  confirm adjusted towercanno location: 
.go xyz 4489.819824 1955.636108 441.996002 571

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Comment by GastonCK** [comment]
Tested, looks good

Before
WoWScrnShot_081223_114736


After
WoWScrnShot_081223_120811

**Comment by heyitsbench** [comment]
May I ask the source on the new coordinates?

**Comment by Kitzunu** [comment]
I want this sniffed. Should not be hard to acquire

File: src/server/game/Battlefield/Zones/BattlefieldWG.h [hunk 1]
```diff
@@ -1041,7 +1041,7 @@ const WintergraspTowerCannonData TowerCannon[WG_MAX_TOWER_CANNON] =
         2,
         {
             { 4448.138184f, 1974.998779f, 441.995911f, 1.967238f },
-            { 4448.713379f, 1955.148682f, 441.995178f, 0.380733f },
+            { 4489.819824f, 1955.636108f, 441.996002f, 0.353242f },
```
Comment (heyitsbench) [inline src/server/game/Battlefield/Zones/BattlefieldWG.h:1044]:
```suggestion
            { 4486.3257f, 1954.6545f, 442.0783f, 0.349065840244293212f },
```
Taken from 51666.
CC: @Kitzunu

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**